### PR TITLE
Drop Python 2 support from master branch

### LINKED ISSUE #1034: Move to Python3 shebangs
Python 2 will be sunset in 2020: https://pythonclock.org

Some modern Linux distributions will not support Python 2 anymore.
Our code is compatible with both Python 2 and 3.
However, the shebangs often contain python2.
The downstream packagers will have to patch these code.

For example, see: https://github.com/OpenSCAP/openscap/blob/maint-1.2/utils/scap-as-rpm#L1

Do you want to change shebangs to `#!/usr/bin/python3`?

Another problem is that we sometimes use "#!/usr/bin/env` which is forbidden by Fedora: https://fedoraproject.org/wiki/Packaging:Guidelines#Shebang_lines

### PR DESCRIPTION
**Pull request by matejak**
Based on #1037. This one is rebased, and contains minor fixes.
This PR tries to move to Python 3 in master branch, drops Python 2 support and sometimes cleans the code up.
The changes include:
* Updating shebangs to #!/usr/bin/python3 (fixes #1034 )
* Update README and User Manuals
* Remove SWIG bindings for Python 2
* Remove fallback solutions, that are now in Python 3 by default, eg. argparse or subprocess.check_output

### CODE REVIEW

**Comment by matejak** [comment]
@mpreisler I have become Jan's partner in crime, so feel free to merge it as I am no longer independent.

File: README.md [hunk 1]
```diff
@@ -37,21 +37,25 @@ cd openscap
 
 2) To build the library you will also need the following build dependencies
 (some of these are optional, if they are not detected, openscap will be compiled
-without respective optional features). On RHEL 7 / Fedora, the command to
```
Comment (mpreisler) [inline README.md]: Why was the note about RHEL7 / Fedora removed? Doesn't seem to be related to python2.

**Comment by mpreisler** [comment]
Found one minor thing that I am curious about, other than that I think this is ready to be merged.

File: README.md (hunk 1 above)
Comment (matejak) [reply README.md]: I thought that as there is only Python2 on RHEL7, I will take the mention about it out. Fedora also installs Python 2 by default.

Comment (mpreisler) [reply README.md]: But the rest of the packages are RHEL / Fedora specific. There is a separate section for ubuntu. It doesn't read to me like the note about RHEL 7 and Fedora is there just because of python2.

Comment (matejak) [reply README.md]: OK, I have put back the sentence with `RHEL / Fedora / CentOS` list.

**Comment by mpreisler** [comment]
ACK
